modules/rostests/apitests/ntdll/RtlAllocateHeap.c

```diff
@@ -9,58 +9,74 @@
 
 #include <pseh/pseh2.h>
 
-PVOID Buffers[0x100];
-
 START_TEST(RtlAllocateHeap)
 {
+    PVOID Buffers[0x100];
     USHORT i;
     HANDLE hHeap;
-    BOOLEAN Aligned = TRUE;
+    BOOLEAN Not8BytesAligned = FALSE;
+    BOOLEAN Not16BytesAligned = FALSE;
     RTL_HEAP_PARAMETERS Parameters = {0};
 
-    for (i = 0; i < 0x100; ++i)
+    for (i = 0; i < ARRAYSIZE(Buffers); ++i)
     {
+        /* RtlAllocateHeap() totally ignores HEAP_CREATE_ALIGN_16 (it's only for RtlCreateHeap() */
```

Comment:
sorry i had a typo in the suggestion, lol ^^
```suggestion
        /* RtlAllocateHeap() totally ignores HEAP_CREATE_ALIGN_16 (it's only for RtlCreateHeap) */
```